frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts

```diff
@@ -121,14 +121,57 @@ export class ChallengesettingsComponent implements OnInit, OnDestroy {
     icon: 'fa fa-toggle-off',
   };  
 
+  /**
+   * Phase split list
+   */
+  phaseSplits = [];
+
+  /**
+   * Phase selection type (radio button or select box)
+   */
+  phaseLeaderboardSelectionType = 'selectBox';
+
+   /**
+    * Select box list type
+    */
+  phaseLeaderboardSelectionListType = 'settingsPhaseSplit';
+
+  /**
+   * Currently selected phase split
+   */
+  selectedPhaseSplit: any = null;
+
+  /**
+   * phase visibility state and it's icon
+   */
+  leaderboardVisibility = {
+    state: 'Private',
+    icon: 'fa fa-toggle-off',
+  };
+
+  /**
+   * If leaderboard of phase split is public
+   */
+   isPhaseSplitLeaderboardPublic: number = 1;
```

Comment:
this should be boolean

Reply:
Actually, phase split visibility is defined by number(`1` or `3`)